Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -511,6 +511,91 @@ unsigned int numBridgeheadAtoms(const RDKit::ROMol &mol,
   return res;
 }
 
+/// Awesome StackOverflow response:
```

Comment:
Huh, this is interesting.
We should think about moving this to RDBoost and using it more generally